seacrowd/sea_datasets/etos/etos.py

```diff
@@ -0,0 +1,206 @@
+
+    UPOS_TAGS = [
+        "NOUN",
+        "PUNCT",
+        "ADP",
+        "NUM",
+        "SYM",
+        "SCONJ",
+        "ADJ",
+        "PART",
+        "DET",
+        "CCONJ",
+        "PROPN",
+        "PRON",
+        "X",
+        "_",
+        "ADV",
+        "INTJ",
+        "VERB",
+        "AUX",
+    ]
+
+    BUILDER_CONFIGS = [
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_source",
+            version=datasets.Version(_SOURCE_VERSION),
+            description=f"{_DATASETNAME} source schema",
+            schema="source",
+            subset_id=f"{_DATASETNAME}",
+        ),
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_seacrowd_seq_label",
+            version=datasets.Version(_SOURCE_VERSION),
+            description=f"{_DATASETNAME} sequence labeling schema",
+            schema="seacrowd_seq_label",
+            subset_id=f"{_DATASETNAME}",
+        ),
+    ]
+
+    def _info(self) -> datasets.DatasetInfo:
+        if self.config.schema == "source":
+            features = datasets.Features(
+                {
+                    "sent_id": datasets.Value("string"),
+                    "text": datasets.Value("string"),
+                    "tokens": datasets.Sequence(datasets.Value("string")),
+                    "lemmas": datasets.Sequence(datasets.Value("string")),
+                    "upos": datasets.Sequence(datasets.features.ClassLabel(names=self.UPOS_TAGS)),
+                    "xpos": datasets.Sequence(datasets.Value("string")),
+                    "feats": datasets.Sequence(datasets.Value("string")),
+                    "head": datasets.Sequence(datasets.Value("string")),
+                    "deprel": datasets.Sequence(datasets.Value("string")),
+                    "deps": datasets.Sequence(datasets.Value("string")),
+                    "misc": datasets.Sequence(datasets.Value("string")),
+                }
+            )
+
+        elif self.config.schema == "seacrowd_seq_label":
+            features = schemas.seq_label_features(self.UPOS_TAGS)
+
+        else:
+            raise NotImplementedError(f"Schema '{self.config.schema}' is not defined.")
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager: datasets.DownloadManager) -> List[datasets.SplitGenerator]:
+        """
+        Returns SplitGenerators.
+        """
+
+        train_path = dl_manager.download_and_extract(_URLS)
+
+        return [
+            datasets.SplitGenerator(
+                name=datasets.Split.TRAIN,
+                gen_kwargs={
+                    "filepath": train_path,
+                    "split": "train",
+                },
+            )
+        ]
+
+    def _generate_examples(self, filepath: Path, split: str) -> Tuple[int, Dict]:
```

Comment:
Remove `split` argument if you don't use it in `_generate_examples`.

Reply:
This is actually common in other dataloaders. The `split` argument is passed here because we put it in the `gen_kwargs` in the previous function. If you want to remove it, don't forget to remove the `split` part in the `gen_kwargs` as well.